tests/run-with-compiler/i3947.scala

```diff
@@ -0,0 +1,24 @@
+
+import scala.quoted._
+import dotty.tools.dotc.quoted.Toolbox._
+
+object Test {
+
+  def main(args: Array[String]): Unit = {
+
+    def test[T](clazz: java.lang.Class[T]): Unit = {
+      val lclazz = clazz.toExpr
```

Comment:
What's the purpose of `lclazz` in the test?

Reply:
Oh, I see. I lifted it directly in the quote. I will remove it or use it instead.

Reply:
Done